SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -244,11 +258,17 @@ void USpatialSender::SendComponentUpdates(UObject* Object, USpatialActorChannel*
 
 void FillComponentInterests(FClassInfo* Info, bool bNetOwned, TArray<Worker_InterestOverride>& ComponentInterest)
 {
-	Worker_InterestOverride SingleClientInterest = { Info->SingleClientComponent, bNetOwned };
-	ComponentInterest.Add(SingleClientInterest);
+	if (Info->SchemaComponents[TYPE_OwnerOnly] != 0)
```

Comment:
Should we typedef this as INVALID_COMPONENT_ID ?

Reply:
Good idea